server_addon/houdini/client/ayon_houdini/plugins/publish/extract_render.py

```diff
@@ -32,6 +33,8 @@ def process(self, instance):
                 rop_node.setParms({"RS_archive_enable": 1})
             elif product_type == "vray_rop":
                 rop_node.setParms({"render_export_mode": "2"})
+            elif product_type == "usdrender":
+                rop_node.setParms({"runcommand": 0})
```

Comment:
Seeing these parms being changes in the extractor and stay that way really scares me. We should usually not at all change the user's workfile - I understand the reasoning, but it goes against all of the concepts of Pyblish and thus also how publishing in OpenPype and then AYON has been structured.

We should really track this as an issue, in this case I'm fine with the logic if it works because it aligns with existing code that's there - but we should really find better ways to do this then 'breaking' the user's scene.

Reply:
I don't like it too.  
Yes, it aligns with existing code that's there... but, I wrote that existing code and I don't like that existing code either. 
That's why I created this issue https://github.com/ynput/ayon-core/issues/417 and made suggested a PR https://github.com/ynput/ayon-core/pull/418 which made the creators more complex.

Reply:
Ah - issue already exists, ignore me!